tests/msc2716_test.go

```diff
@@ -1126,16 +1126,6 @@ func TestImportHistoricalMessages(t *testing.T) {
 	})
```

Comment:
Looks like the Dendrite CI job is [failing on `main`](https://github.com/matrix-org/complement/actions/runs/3482537405/jobs/5824967391) as well